American Whitewater/ViewControllers/Reaches/RunsListViewController.swift

```diff
@@ -411,8 +341,28 @@ extension RunsListViewController: UITableViewDelegate, UITableViewDataSource {
     
     
     func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
-        // handle the case when a filter shows 0 items
-        if indexPath.section == 1 && (fetchedResultsController?.fetchedObjects?.count ?? 0) == 0 {
+        // handle the case when there are no fetched objects
+        if
+            indexPath.section == 1,
+            (fetchedResultsController?.fetchedObjects?.count ?? 0) == 0
+        {
+            if totalReachCount == 0 {
+                // The loading state.
+                // There aren't any reaches stored locally, so (presumably?) we must be doing an initial load of reaches from the network
```

Comment:
Pull to refresh should still work here right? So even if we aren't currently loading, this is user resolvable.